tools/validators/instance_validator/tests/fake_instances/GOOD/code_with_spaces.yaml

```diff
@@ -14,7 +14,7 @@
 SDC_EXT 2-1 / Rm 2D2-GUID:
   type: HVAC/SDC_EXT
   code: SDC_EXT-2-1
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
done